### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by fregante**
😎

### CODE REVIEW

File: source/features/highest-rated-comment.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ function linkBestComment(bestComment: HTMLElement): void {
 	}
 
 	const text = select('.comment-body', bestComment)!.textContent!.slice(0, 100);
-	const {hash} = select<HTMLAnchorElement>('a.js-timestamp', bestComment)!; // TODO: Drop generic once TypeScript sorts it out, it shouldn't be needed
+	const {hash} = select('a.js-timestamp', bestComment)!;
```
Comment (fregante) [inline source/features/highest-rated-comment.tsx:74]: I knew I was right!

Comment (yakov116) [reply source/features/highest-rated-comment.tsx:74]: As always!

File: source/features/linkify-user-location.tsx [hunk 2]
```diff
@@ -1,35 +1,29 @@
...
 }
 
-function addLocation({nextElementSibling, nextSibling}: SVGElement): void {
-	attachElement(
-		// `nextSibling` alone might point to an empty TextNode before an element, if there’s an element
-		nextElementSibling ?? nextSibling as Element,
-		{forEach: linkify},
-	);
-}
-
```
Comment (fregante) [inline source/features/linkify-user-location.tsx:32]: Change tested, it works

File: source/features/quick-review.tsx [hunk 3]
```diff
@@ -6,26 +6,19 @@ import delegate, {DelegateEvent} from 'delegate-it';
...
+		</span>,
 	);
 }
 
-function addSidebarReviewButton(reviewersSection: Element): void {
-	attachElement(reviewersSection, {
-		append: createReviewLink,
-	});
-}
-
```
Comment (fregante) [inline source/features/quick-review.tsx:28]: Change tested, it works

File: source/helpers/types.d.ts [hunk 4]
```diff
@@ -4,14 +4,15 @@ import type {ParseSelector} from 'typed-query-selector/parser';
...
 		querySelectorAll<S extends string>(
-			selector: S,
+			selector: S | readonly S[],
 		): NodeListOf<ParseSelector<S, HTMLElement>>;
 	}
 
 	interface Element {
-		closest<S extends string>(selector: S): ParseSelector<S, HTMLElement> | null;
+		closest<S extends string>(selector: S | readonly S[]): ParseSelector<S, HTMLElement> | null;
+		matches(selectors: string | readonly string[]): boolean;
```
Comment (fregante) [inline source/helpers/types.d.ts:16]: `readonly` allows passing arrays like `['input', 'textarea'] as const`, which are tighter than `string[]`

File: package.json [hunk 5]
```diff
@@ -42,7 +42,7 @@
 	},
 	"dependencies": {
 		"@cheap-glitch/mi-cron": "^1.0.1",
-		"@primer/octicons-react": "^17.5.0",
+		"@primer/octicons-react": "^17.11.1",
```
Comment (fregante) [inline package.json]: I had to undo this because dom-chef started failing 🥲

**Pull request merged**